index.html

```diff
@@ -89,48 +97,142 @@
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform','translate(' + margin.left + ',' + margin.top + ')');
+   
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([d3.min(data,function (d){ return d.cfScore;}), Math.ceil(d3.max(data, function (d){ return d.cfScore; } )) ])
+        .range([margin.left,(margin.left + width)])
+        .nice();
+
+console.log(Math.ceil(d3.max(data, function (d){ return d.cfScore; } )))
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d){ return d.distPart; } ))
+      .range([(height-margin.top),0])
+      .nice();    
+   
     // AXES
-
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .tickSize(-height)
+      .orient("bottom");
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .tickSize(-width)
+      .orient("left");
+    svg.append("g")
+      .classed("x axis", true)
+      .attr("transform", "translate(0," + (height-margin.top) + ")")
+      .call(xAxis);
+    svg.append("g")
+      .classed("y axis", true)
+      .attr("transform","translate(" + margin.left + ",0)")
+      .call(yAxis);
+    var line = d3.svg.line()
+      .x(function(d) { return xScale(d.x); })
+      .y(function(d) { return yScale(d.y); });
+    svg.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate(0,"+(height/2)+")rotate(-90)")
+        .text("District Partisanship");
+    svg.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate("+ (width/2) +","+(height + margin.top + (margin.bottom/2))+")")
+        .text("Candidate CF Score");
     // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
+    d3.select('#party').on('change', function () {
+      options.party = d3.event.target.value;
       update();
     });
   }
-
   function update() {
     // TRANSFORM
     txData = data.slice();
-
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
+    if (options.party !== 'all') {
+      var party = options.party;
       txData = txData.filter(function (d) {
-        return d.year >= yearRange[0] && d.year <= yearRange[1];
+        return d.party === party;
       });
     }
-
     // SCATTERPLOT
-
+    var points = svg.selectAll('.point')
+      .data(txData);
+
+    points.enter().append('circle')
+      .classed("point", true)
+      .attr('r', 5);
+    
+    points
+      .classed('point',true)
+      .classed("R", function (d) {return d.party === "rep";})
+      .classed("D", function (d) {return d.party === "dem";})
+      .attr('cx', function (d){return x(d.cfScore) ;})
+      .attr('cy', function (d){return y(d.distPart) ;});
+
+    points.exit().remove();
+    
     // HISTOGRAMS
+    // X-HIST
+ 
+    var xdata = data.map(function(d) { return d.cfScore; });
```

Comment:
Note that you're not using the transformed data here, so it won't respond to changes in the filter.